### PR TITLE
[Fixes #235] Add email templates in authv1 app

### LINKED ISSUE #235: Add Email templates in authv1 app
## Feature Request

Is your feature request related to an existing [Issue](https://www.github.com/Auto-DL/Generator/issues) or a new implementation?
No

### Feature Description
Create a new file in authv1 app to store email templates.

### PR DESCRIPTION
**Pull request by mahekn23**
## Pull Request

fixes #235 

### Description
New file added in authv1 app (emails.py) for storing email templates

### Checklist:
- [x] Followed the guidelines mentioned in our [Contributing document](../CONTRIBUTING.md)
- [x] Ensured that there aren't other open [Pull Requests](https://www.github.com/Auto-DL/Generator/pulls) for the same update/change
- [x] This PR is linked to an [Issue](https://www.github.com/Auto-DL/Generator/issues) and the issue number has been mentioned in the title and body of the PR
- [x] The code is clean and docstrings have been added or updated as required
- [x] Code is linted/formatted locally prior to submission (using `black`)

Signed-off-by: mahekn23 <2018.mahek.nagdev@ves.ac.in>

### CODE REVIEW

File: BackEndApp/authv1/emails.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    def __init__(self, user, request):
+        self.user = user
+        self.request = request
+
+    def update_password(self, username, email, otp):
+        subject = "Auto-DL Request to Update Password"
+        msg = (
+            "You ("
+            + username
+            + ") requested to update your password. Click here to reset your password "
```
Comment (shintan777) [inline BackEndApp/authv1/emails.py:15]: We are not doing the link right. Please change the messgae

Comment (mahekn23) [reply BackEndApp/authv1/emails.py]: yesss just edited that....please check

File: BackEndApp/authv1/emails.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+from django.core.mail import send_mail
+from BackEndApp.settings import EMAIL_HOST_USER
+
+
+class Email_Templates:
+    def __init__(self, user, request):
+        self.user = user
+        self.request = request
+
+    def update_password(self, username, email, otp):
```
Comment (ADI10HERO) [inline BackEndApp/authv1/emails.py:10]:
```suggestion
    def forgot_password(self, username, email, otp):
```

The template is for forgot password right?

File: BackEndApp/authv1/emails.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        subject = "Auto-DL Request to Update Password"
+        msg = (
+            "You ("
+            + username
+            + ") requested to update your password. "
+            + "Your OTP is: "
+            + otp
+        )
+        send_mail(subject, msg, EMAIL_HOST_USER, [email])
```
Comment (ADI10HERO) [inline BackEndApp/authv1/emails.py:19]:
```suggestion
```

Please do not send email from this file. There's a reason it's called "EmailTemplates" right ? :)

**Review by ADI10HERO — CHANGES_REQUESTED** [review]

File: BackEndApp/authv1/emails.py (hunk 3 above)
Comment (mahekn23) [reply BackEndApp/authv1/emails.py]: Made the necessary changes

**Review by ADI10HERO — APPROVED** [review]

**Pull request merged**